### DISCUSSION TITLE
[0,0,0,0] boxes leaking?

### DISCUSSION THREAD

**Discussion by Nekronik** [vladmandic/human#240] [Q&A] [answered]
Hello, I noticed that in some situations persons are detected where the face has a box of `[0,0,0,0]`

Correct me if I am wrong but, I think a box of area 0 means that there is no face and should not be included in the results.

I spotted that these 0-area-boxes appear in at least two situations:
1) At the same pace of `skipTime` (I have it set to 1000ms)
![image](https://user-images.githubusercontent.com/7140454/149510703-b12486c7-5f28-4b1d-9ebb-254fcbd46f36.png)
2) When nothing is detected, as many as `maxDetected` 0-area-boxes are returned (I have it set as 3 for the example)
![image](https://user-images.githubusercontent.com/7140454/149510544-c8cd7c33-d494-41a6-9220-e03ca5f0e9ea.png)

I noticed that the results rate when nothing is detected is slower and I guess this is because it tries to spend more time detecting anything when nothing is found.

For the record:
- I spotted this behavior in versions `2.5.1` and `2.5.7` (did not test it in any more than this)
- nothing else (i.e. gestures) apart from a 0-area-box is inside any of these "persons":
![image](https://user-images.githubusercontent.com/7140454/149511304-591df089-4b15-48a9-aa0c-b3755f9f765f.png)

Is this the expected behavior? I did not know if I had to open this as an issue or discussion.

**Comment by vladmandic** [comment, marked as answer]
definitely a bug, i've created issue #241 to track it.
